build/teamcity/cockroach/nightlies/microbenchmark_weekly.sh

```diff
@@ -61,17 +73,21 @@ for rev in "${revisions[@]}"; do
   sha_arr+=("$sha")
 done
 
+# Check if the baseline cache exists and copy it to the output directory.
+baseline_cache_path="gs://$BENCH_BUCKET/cache/$GCE_MACHINE_TYPE/$SANITIZED_BENCH_PACKAGE/${sha_arr[1]}"
+if gsutil ls "$baseline_cache_path" &>/dev/null; then
```

Comment:
I was going to suggest using `check_gcs_path_exists`, but it seems that the support file is not `source`d here. I actually can't seem to find where that file is used. 🤔

Reply:
I can probably move this to support, so that it can be reused, although it's such a small function, but probably makes it more clear what it's doing.

Reply:
Nah, I get it now, I was grepping for the wrong string.